clang/examples/DPCT/Runtime/cudaMemcpy2DFromArray.cu

```diff
@@ -1,8 +1,9 @@
-void test(void *pv, size_t s1, cudaArray_const_t a, size_t s2, size_t s3,
-          size_t s4, size_t s5, cudaMemcpyKind m) {
+void test(void *dst, size_t s1, size_t s2, size_t s3, size_t s4, size_t s5) {
+  // TODO: src's type need to be changed to cudaArray_const_t
```

Comment:
fix the todo

Reply:
Now the cudaArray_const_t cannot be migrated correctly.